src/cascadia/TerminalApp/CascadiaSettingsSerialization.cpp

```diff
@@ -36,6 +37,8 @@ static constexpr std::string_view DisabledProfileSourcesKey{ "disabledProfileSou
 
 static constexpr std::string_view Utf8Bom{ u8"\uFEFF" };
 static constexpr std::string_view DefaultProfilesIndentation{ "        " };
+static constexpr std::string_view SettingsSchemaFragment{ "\n"
+                                                          R"(    "$schema": "https://aka.ms/terminal-profiles-schema",)" };

```

Comment:
My regex monstrosity is alive!

Reply:
you calling out this line taught me that we will almost certainly corrupt a user's profile if we hardcode then `,` here. More thought is required. :grin: